s12/protobuf/proto/validator.proto

```diff
@@ -196,7 +196,8 @@ message IdRules {
   optional string version = 2 [default = "v4"];
 
   // Also allow legacy_id format, similar to validator.legacy_id
-  // Normal UUID validation will be attempted first, if it fails it will fall back to IsLegacyID method
+  // Normal UUID validation will be attempted first, i
+  // If it fails it will fall back to IsLegacyID method, this will check for long UUID or S12ID
```

Comment:
Wouldn't it be better to explicitly enable S12 ID support if it is required? This could be done by adding another option like `optional bool s12id = 4 [default = false];` I can imagine that in some cases you do not want to consider S12 IDs valid which then requires further validation in the business logic.

Reply:
initially that was my approach. But the current implementation assumes the UUIDv is "4" by default. Then the validation string would look  like "uuidv4 = "", s12id = true". Found it little bit unfriendly